server/utils.go

```diff
@@ -19,15 +19,24 @@ import (
 	"unicode"
 	"unicode/utf8"
 
+	"github.com/nyaruka/phonenumbers"
 	"github.com/tinode/chat/server/auth"
 	"github.com/tinode/chat/server/store/types"
-	"github.com/nyaruka/phonenumbers"
 
 	"golang.org/x/crypto/acme/autocert"
 )
 
-var tagPrefixRegexp = regexp.MustCompile(`^([a-z]\w{0,5}):\S`)
-var alnumPrefixRegexp = regexp.MustCompile(`^[[:alnum:]]+:\S`)
+// Tag with prefix:
+// * prefix starts starts with an ASCII letter, contains ASCII letters, numbers, from 2 to 16 chars.
+// * tag body starts and ends with a non-space Unicode character, has 2 or more characters, otherwise not restricted.
+var tagPrefixRegexp = regexp.MustCompile(`^([a-z]\w{1,15}):(\S.*\S)$`)
```

Comment:
Maybe rename prefixedTagRegexp for better readability?

Reply:
done